org.eevolution.warehouse/src/main/java/base/org/eevolution/process/GenerateShipmentOutBound.java

```diff
@@ -110,7 +111,7 @@ private void createShipment(MWMInOutBoundLine outboundLine) {
         // Generate Shipment based on Outbound Order
         if (outboundLine.getC_OrderLine_ID() > 0) {
             MOrderLine orderLine = outboundLine.getOrderLine();
-            if (orderLine.getQtyOrdered().subtract(orderLine.getQtyDelivered()).subtract(outboundLine.getPickedQty()).signum() <= 0 && !isIncludeNotAvailable())
+            if (orderLine.getQtyOrdered().subtract(getQtyTotalPicked(outboundLine.getC_OrderLine_ID())).signum() < 0 && !isIncludeNotAvailable())
```

Comment:
Hi @e-Evolution you should validate for delivered quantity also, see:

```Java
if (orderLine.getQtyOrdered().subtract(orderLine.getQtyDelivered()).subtract(outboundLine.getQtyTotalPicked(outboundLine.getC_OrderLine_ID())).signum() < 0 && !isIncludeNotAvailable())
				return;
```
Best regards